src/main/resources/static/front/css/common/low-posture.css

```
@@ -109,6 +109,20 @@ html.posture-low .ai-chat-panel--open { transform: translateY(0); }
 /* 하단 시트(대화기록) 열리면 ♿ 토글 숨김 — 패널 하단 버튼과 겹침 방지 */
 html.posture-low body:has(.ai-chat-panel--open) .lpost-toggle { display: none; }
 
+/* 음성 대화 패널(.vc-chat — 결제 등 채팅 UI 없는 화면 공용, voice-controller.js 주입):
+   기본은 우측 풀높이 드로어 → 저자세에선 .ai-chat-panel 과 동일하게 하단 시트(절반)로 */
+html.posture-low .vc-chat {
+    top: auto;
+    inset: auto 0 0 0;
```

Stylelint (17.12.0)

[error] Overridden property "top" by shorthand "inset" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)